### PR TITLE
admin_signup

### PR DESCRIPTION
**Pull request by praveen021097**
- admin signup page 
- captcha 
-

### CODE REVIEW

File: src/Pages/AdminSignup.jsx [hunk 1]
```diff
@@ -1,12 +1,183 @@
...
-  )
+      const [captcha,setCaptcha] =useState("");
+      const [userCaptcha,setUserCaptcha] =useState(false);
+      const captchaRef = useRef();
+
+  const captchaCode =()=>{
+    const alpha =['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V'
+    ,'W','X','Y','Z','1','2','3','4','5','6','7','8','9','0','a','b','c','d','e','f','g','h','i',
+    'j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z', '!','@','#','$','%','^','&','*','+'];
+
```
Comment (mangesh0712) [inline src/Pages/AdminSignup.jsx:15]: as alpha seems like a constant, plz add it in constant file and import it

File: src/Pages/AdminSignup.jsx [hunk 2]
```diff
@@ -1,12 +1,183 @@
...
+    }
+
+    useEffect(()=>{
+      captchaCode();
+
+    },[])
+const handleRefreshCaptcha =()=>{
+  captchaCode();
+  setUserCaptcha(false)
+
```
Comment (mangesh0712) [inline src/Pages/AdminSignup.jsx:35]: format file plz , remove unnecessary spaces

File: src/Pages/AdminSignup.jsx [hunk 3]
```diff
@@ -1,12 +1,183 @@
...
+}
+const matchcaptcha =(e)=>{
+    const userCap = e.target.value;
+    if(captcha===userCap && userCap.length==6){
+      setUserCaptcha(true)
+
```
Comment (mangesh0712) [inline src/Pages/AdminSignup.jsx:41]: remove unnecessary spaces

File: src/Pages/AdminSignup.jsx [hunk 4]
```diff
@@ -1,12 +1,183 @@
...
...2 30.0082 12.4292C30.9156 12.3552 31.809 12.4327 32.6745 12.7203C34.2743 13.2489 35.2586 14.3786 35.8233 15.9009C36.229 16.9944 36.3426 18.1362 36.3444 19.2917C36.3478 21.9815 36.3461 24.6714 36.3461 27.3613C36.3461 27.4284 36.3374 27.4973 36.3304 27.592C35.1082 27.592 33.8914 27.592 32.6325 27.592C32.6325 27.4577 32.6325 27.3251 32.6325 27.1925C32.6325 24.6955 32.6465 22.2002 32.622 19.7032C32.6168 19.1298 32.5486 18.5374 32.3877 17.9881C32.0573 16.8635 31.281 16.142 30.0781 16.0077C28.7161 15.8561 27.557 16.2436 26.8069 17.4628C26.4397 18.0587 26.2579 18.7199 26.2509 19.4122C26.2247 21.9867 26.2142 24.5612 26.2002 27.1374C26.2002 27.2838 26.2002 27.4301 26.2002 27.5989C24.9414 27.5989 23.7105 27.5989 22.4394 27.5989C22.4394 27.456 22.4394 27.3216 22.4394 27.189C22.4377 24.5216 22.4447 21.8541 22.4254 19.1866C22.4184 18.3256 22.1929 17.5076 21.5932 16.8515C20.6631 15.8338 19.4864 15.7993 18.2748 16.1799C16.8865 16.6156 16.2764 17.6815 16.1155 19.0368C16.0596 19.5104 16.0578 19.9925 16.0561 20.4696C16.0508 22.7031 16.0543 24.9383 16.0543 27.1718C16.0543 27.3062 16.0543 27.4422 16.0543 27.6213C14.7832 27.6213 13.5366 27.6213 12.2918 27.6213C12.29 22.648 12.29 17.6781 12.29 12.7082Z"
```
Comment (mangesh0712) [inline src/Pages/AdminSignup.jsx:61]: are you using any lib for captha ? if not plz use it as this this is not maintainable

File: src/Pages/AdminSignup.jsx [hunk 5]
```diff
@@ -1,12 +1,183 @@
...
+            <div className=" flex flex-col text-left  mx-[20px]">
+              <label
+                htmlFor=""
+                className="block font-medium text-sm text-gray-700"
+              >
+                Name
+              </label>
+              <input
+                type="text"
+                className=" border border-gray-300 outline-none focus:border-indigo-200 focus:bg-indigo-50 focus:ring focus:ring:indigo-100 focus:ring-opacity-50 h-10 rounded-md shadow-sm mt-1 w-full"
```
Comment (mangesh0712) [inline src/Pages/AdminSignup.jsx:107]: format file plz

File: src/Pages/AdminSignup.jsx (hunk 1 above)
Comment (praveen021097) [reply src/Pages/AdminSignup.jsx]: i removed it

File: src/Pages/AdminSignup.jsx (hunk 2 above)
Comment (praveen021097) [reply src/Pages/AdminSignup.jsx]: removed

File: src/Pages/AdminSignup.jsx (hunk 5 above)
Comment (praveen021097) [reply src/Pages/AdminSignup.jsx]: formated now

File: src/Pages/AdminSignup.jsx (hunk 3 above)
Comment (praveen021097) [reply src/Pages/AdminSignup.jsx]: removed

**Pull request closed without merging**